src/charactersheet/viewmodels/common/characters/index.js

```diff
@@ -17,26 +17,17 @@ export function CharactersViewModel(params) {
     self.characters = ko.observableArray([]);
     self.componentStatus = params.modalStatus || ko.observable(false);
     self.modalStatus = ko.observable(false);
+    self.selectedCharacter = ko.observable();
 
-    self.load = function() {
+    self.load = () => {
         self.characters(PersistenceService.findAll(Character));
         self.modalStatus(self.componentStatus());
-    };
-
-    self.changeCharacter = function(character) {
```

Comment:
While I do appreciate your enthusiasm with improving our code coverage, I don't believe this should be achieved by removing functionality. Changing characters is impossible in this branch because this method was removed. I'm surprised you're not getting an error because of it.

Reply:
:P I was able to change characters. Plus I couldn't find any place where that method was even called. The switcher just calls the Character Manager to do the switch not itself.

Reply:
Try it in Chrome or clear your cache. It doesn't work for me. I got your latest code and everything is fresh.

Reply:
My bad. I was just deleting characters and that caused the switch. Good catch.